protocols/virtualfund/virtualfund.go

```diff
@@ -246,7 +245,7 @@ func constructFromState(
 		init.ToMyRight = &Connection{}
 
 		if consensusChannelToMyRight == nil {
-			return Objective{}, fmt.Errorf("non-bob virtualfund objective requires non-nil ledger channel")
```

Comment:
Seems like out of scope, but I will let you off :-P

Reply:
Indeed. One of those commits that one forgets about...